docs/sbom-tool-api-reference.md

```diff
@@ -1,16 +1,16 @@
 # SBOM API Reference
 
-The SBOM tool can be called using a C# API. This guide will help you integrate the SBOM tool API package in your .NET project.
+Users can use the C#-based SBOM API for calling the SBOM tool. This guide is intended to assist users in integrating the SBOM tool API package in a .NET project.
 
 ## Prerequisites
-* You have a .NET project that can ingest packages from nuget.org.
-* Only projects that target .NET 6 or higher are supported, we don't have a .NET Framework implementation for the SBOM API. 
-* You are using [Microsoft Dependency Injection](https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection) in your project.
-* Add the **SBOMToolsPublic** repository to your nuget.config, you can check the steps to get it added to your project by clicking the **'Connect to Feed'** button on the feed page [here](https://dev.azure.com/mseng/PipelineTools/_artifacts/feed/SBOMToolsPublic)
+
+* A .NET project that can ingest packages from nuget.org.
+* Only projects that target .NET 6 or higher.  This API currently provides no support for implementation of .NET Framework for the SBOM API. 
```

Comment:
2 spaces after the period. It looks like this happens in several places. If I'm wrong feel free to just resolve the comments.